lib/salesforce_ar_sync/wrappers/restforce.rb

```diff
@@ -0,0 +1,21 @@
+# We'll expose a public interface that only contains the methods we need here
+module SalesforceArSync
+  module Wrappers
+    class Restforce
+      attr_accessor :sf_client
+      def initialize(sf_client)
+        @sf_client = sf_client
+      end
+
+      # def find(sobject, id, field = nil)
+      #   @sf_client.find(sobject, id, field)
+      # end
+
+      def http_post; end
```

Comment:
are these just to keep old `SF_CLIENT.http_post` calls from blowing up?

Reply:
aw damnit I didn't realize that made it's way in, decided this effort was going to the website since not much is going on with AR sync & databasedotcom